openmdao/visualization/connection_viewer/viewconns.py

```diff
@@ -36,7 +36,7 @@ def _val2str(val):
 def view_connections(root, outfile='connections.html', show_browser=True,
                      show_values=True, precision=6, title=None):
     """
-    Generate a self-contained html file containing a detailed connection viewer.
+    Generate a self-contained html (or csv) file containing a detailed connection viewer.
```

Comment:
Having "self-contained" followed by "containing" seems redundant. I know you didn't add it, but I would remove it.